setup.py

```diff
@@ -96,6 +96,8 @@ def get_git_sha():
         'thrift-sasl>=0.2.1',
         'unicodecsv',
         'unidecode>=0.04.21',
+        'croniter==0.3.24',
+        'selenium==3.12.0',
     ],
```

Comment:
Nit: is there a reason why these are pegged to these specific versions, and not `<=` or `>=`? The version of `croniter` current, but selenium has had two minor releases since, with the latest yesterday. If there are no known compatibility issues with future versions, perhaps these should be left unpegged in `setup.py`.

Reply:
@villebro thank you for the comment. It was mostly for a safe bet, to avoid breakages. Specially with selenium. I wanted to peg it to a version which I knew worked. Selenium versions and some webdrivers end up having compatibility issues. I am OK with leaving it unpegged, but I don't think our testing is good enough to catch future breakages.

Reply:
@mahendra IMO fixing deprecations and possible future breakages tends to be fairly straight forward, and shouldn't be a big issue. So from a maintainability perspective leaving them unpegged would probably be better, as it would allow for features/fixes to flow in automatically.

Reply:
@villebro Noted. I just updated the code and removed the pinning. Pushed it to github.

To clarify, my worry was not about the ability to fix breakages. Rather it was to avoid breaking deploys. Specifically when someone installs/upgrades to superset==0.27.0 and their setup script accidentally upgrades selenium to an incompatible version. This could leave them with a broken deploy. We could fix it once we get bug reports.

Reply:
@mahendra I understand, perhaps @betodealmeida or one of the official maintainers can weigh in? As there doesn't seem to be any known incompatibility issues with `croniter>0.3.24` and `selenium>3.12.0`, and this is a new feature, I think any users testing it will be aware that they should expect a bug or two as this is rolled out. If incompatibility issues do arise, I feel they are best taken care of by making changes to the Superset code to accomodate API changes etc. Only in the case of a serious divergence in features or known bugs should we pin to an old version in `master`. Perhaps even fixing the dependency versions in the official setup.py of 0.27 might be an option.

Reply:
@villebro agree with you. I have unpinned the versions. We can fix it in the future if required.

@betodealmeida thoughts?